examples/portworx/variables.tf

```diff
@@ -145,4 +145,39 @@ variable secret_type {
   description = "secret type"
   type        = string
   default     = "k8s"
-}
\ No newline at end of file
+}
+
+##############################################################################
+# Cloud Drive Variables
+##############################################################################
+variable cloud_drive {
+  description = "cloud drive support enabled"
+  type  = string
+  default = "Yes"
+}
+variable num_cloud_drives{
+  description = "No of drives to provisoned(max=3)"
+  type=number
+  validation {
+    condition=var.num_cloud_drives>0 && var.num_cloud_drives<4
+    error_message = "Number of drives should be maximum 3"
```

Comment:
Would it be better to reword this message to `Maximum number of cloud drives that can be provisioned is 3`?